### ISSUE TITLE
Changing the value of the Switch using spacebar scrolls the page down

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1311]
Focusing and pressing the spacebar to change the Switch value forces the page to scroll:
[https://stackblitz.com/edit/react-7sqpid-1zjtcr?file=app%2Fmain.jsx](https://stackblitz.com/edit/react-7sqpid-1zjtcr?file=app%2Fmain.jsx)

Although that this is normal browser behavior, since it is an input element, which should prevent the scrolling internally. 
I have included a possible temporary workaround in the example:
```
      ref.current.element.addEventListener('keydown', (e) => {
        if (e.keyCode == 32) {
         e.preventDefault();
        }
      });
```

**Comment by ivayloandonovprogress** [comment]
The issue has been fixed in [v5.6.0-dev.202208040930](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#ccd09838-74c6-5690-a9e1-15498bba4a3c)

See the following article about obtaining the development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

**Comment by ivayloandonovprogress** [comment]
The issue has been fixed in [v5.6.0-dev.202208090813](https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/#6ca74564-d278-5679-95ea-b88579a59b69)

Demo: https://stackblitz.com/edit/react-7sqpid-7cm1qq?file=app/main.jsx

See the following article about obtaining the development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.
